virt/virtual_machines/virt-delete-vms.adoc

```diff
@@ -4,8 +4,7 @@ include::modules/virt-document-attributes.adoc[]
 :context: virt-delete-vms
 toc::[]
 
-You can delete a virtual machine from the web console or by using the `oc`
-command-line interface.
+You can delete a virtual machine from the web console or by using the `oc` command-line interface.
```

Comment:
s/command-line/command line

Reply:
Change made.

Reply:
Still present in most recent commit

Reply:
Removed the hyphen.

Reply:
Victim of the GitHub outage. Change had been made but not caught by GitHub after the push.

Reply:
From the .adoc file:
[id="virt-delete-vms"]
= Deleting virtual machines
include::modules/virt-document-attributes.adoc[]
:context: virt-delete-vms
toc::[]

You can delete a virtual machine from the web console or by using the `oc` command line interface.